src/test/compile-fail/cast-rfc0401.rs

```diff
@@ -59,10 +59,12 @@ fn main()
     //~^ ERROR casting
     //~^^ HELP through a usize first
     let _ = 3_i32 as bool;
-    //~^ ERROR cannot cast as `bool`
+    //~^ ERROR cannot cast as `bool` [E0054]
```

Comment:
Ah okay. Sure, if you want to match them that way is fine.  You can also leave the `[0054]` off, and it will also succeed.